src/main/resources/hudson/plugins/jobConfigHistory/JobConfigHistory/config.jelly

```diff
@@ -1,54 +1,45 @@
+      <f:textbox/>
     </f:entry>
     <f:advanced>
-      <f:entry title="${%Max number of history entries to keep}" help="/plugin/jobConfigHistory/help/help-maxHistoryEntries.html">
-        <f:textbox name="maxHistoryEntries" value="${it.maxHistoryEntries}"
-           checkUrl="'${rootURL}/plugin/jobConfigHistory/checkMaxHistoryEntries?value='+escape(this.value)"/>
-      </f:entry>
-      <f:entry title="${%Max number of days to keep history entries}" help="/plugin/jobConfigHistory/help/help-maxDaysToKeepEntries.html">
-        <f:textbox name="maxDaysToKeepEntries" value="${it.maxDaysToKeepEntries}"
-           checkUrl="'${rootURL}/plugin/jobConfigHistory/checkMaxDaysToKeepEntries?value='+escape(this.value)"/>
-      </f:entry>
-      <f:entry title="${%Max number of history entries to show per page}" help="/plugin/jobConfigHistory/help/help-maxEntriesPerPage.html">
-        <f:textbox name="maxEntriesPerPage" value="${it.maxEntriesPerPage}"
-           checkUrl="'${rootURL}/plugin/jobConfigHistory/checkMaxEntriesPerPage?value='+escape(this.value)"/>
-      </f:entry>
-      <f:entry title="${%Configuration exclude file pattern}" help="/plugin/jobConfigHistory/help/help-excludePattern.html">
-        <f:textbox name="excludePattern" value="${it.excludePattern}" default="${it.defaultExcludePattern}"
-           checkUrl="'${rootURL}/plugin/jobConfigHistory/checkExcludePattern?value='+escape(this.value)"/>
-      </f:entry>
-      <f:entry help="/plugin/jobConfigHistory/help/help-skipDuplicateHistory.html">
-        <f:checkbox name="skipDuplicateHistory"
-            title="${%Do not save duplicate history}"
-            checked="${it.skipDuplicateHistory}"/>
-      </f:entry>
-      <f:entry help="/plugin/jobConfigHistory/help/help-saveModuleConfiguration.html">
-        <f:checkbox
-            title="${%Save Maven module configuration changes}"
-            name="saveModuleConfiguration"
-            checked="${it.saveModuleConfiguration}"/>
-      </f:entry>
-      <f:entry title="${%Show build badges}" help="/plugin/jobConfigHistory/help/help-showBuildBadges.html">
-        <f:radio name="showBuildBadges" title="${%Never}" value="never" checked="${it.getShowBuildBadges() == 'never'}"/>
-        <br />
-        <f:radio name="showBuildBadges" title="${%Always}" value="always" checked="${it.getShowBuildBadges() == 'always'}"/>
-        <br />
-        <f:radio name="showBuildBadges" title="${%Only for users with configuration permission}" value="userWithConfigPermission" checked="${it.getShowBuildBadges() == 'userWithConfigPermission'}"/>
-        <br />
-        <f:radio name="showBuildBadges" title="${%Only for administrators}" value="adminUser" checked="${it.getShowBuildBadges() == 'adminUser'}"/>
-      </f:entry>
-      <f:entry title="${%Exclude users}" help="/plugin/jobConfigHistory/help/help-excludedUsers.html">
-        <f:textbox name="excludedUsers" value="${it.excludedUsers}" />

```

Comment:
why this value was removed? I'm asking out of curiosity

Reply:
What exactly do you think was removed?
The entry for "excluded users" was simplified and is still present in lines 37-39.
It is only required to define the `field` attribute, the rest is derived if not explicitly set (see [here](https://github.com/jenkinsci/jenkins/blob/jenkins-2.381/core/src/main/resources/lib/form/entry.jelly#L38-L42)).